kalite/distributed/management/commands/test.py

```diff
@@ -29,5 +29,33 @@ class Command(TestCommand):
             dest='no_bdd',
             default=False,
             help="Don't run the behave tests."
-        )
+        ),
+        make_option(
+            '--test-cli',
+            action='store_true',
+            dest='test_cli',
+            default=False,
+            help=("Don't execute tests, just test out that management commands are parsed correctly."
+                  "Needs to be used in conjunction with other cl arguments.")
+        ),
+        make_option(
+            '-z', '--fake-arg',
+            action='store',
+            dest='fake_arg',
+            help="Echo the argument. Used in conjunction with --test-cli to test cl arguments are parsed well."
+        ),
+        make_option(
+            '-y', '--fake-arg2',
```

Comment:
Rather than tack on other options to the `test` command, why not just create another management command specifically testing our CLI options? It seems like these new options aren't about changing the behaviour of the `test` command anyway, and only work with each other.

That's with the assumption that the incorrect parsing happens across all management commands.